README.md

```diff
@@ -18,10 +18,11 @@ using NLayer;
 Then create an `MpegFile`, pass a file name or a stream to the constructor, and use `ReadSamples` for decoding the content:
 
 ```cs
+const int readBufferSize = 44100; // 1 second for mono
```

Comment:
Maybe changing the comment to samples per second times channel count would be clearer. Also, please move the comment to a new line above this line.

`readBufferSize` is probably also a confusing name for this.